docs/changes.txt

```diff
@@ -5,6 +5,7 @@
 ##
 
 - Added hash-mode  2501 = WPA/WPA2 PMK
+- Added hash-mode  1470 = sha256(sha256(password))
```

Comment:
Please use the same format that we used for other hash modes i.e.:
- Added hash-mode  1470 = sha256(sha256($pass))